lib/mkmf.rb

```diff
@@ -2633,7 +2634,8 @@ def create_makefile(target, srcprefix = nil)
     if srcs.any?(&%r"\.(?:#{CXX_EXT.join('|')})\z".method(:===))
       link_so = link_so.sub(/\bLDSHARED\b/, '\&XX')
     end
-    mfile.print link_so, "\n\n"
+    mfile.print link_so, "\n"
+    mfile.print "\t$(Q) $(POSTLINK)\n\n"
```

Comment:
Also you can append `$(POSTLINK)` to `LINK_SO` in configure.ac.

Reply:
Ah, I didn't notice `LINK_SO` can be multiple commands. Looks like we should generalize the way darwin platforms do here. https://github.com/ruby/ruby/blob/a465393cb9b6012f01a98f3da861c92be54445e7/configure.ac#L1110-L1113